java/src/jmri/jmrit/display/layoutEditor/LayoutConnectivity.java

```diff
@@ -199,6 +199,20 @@ public PositionablePoint getAnchor() {
         return anchor;
     }
 
-    private final static Logger log = LoggerFactory.getLogger(LayoutConnectivity.class.getName());
+    @Override
+    public boolean equals(Object o) {
+        boolean result = false; // assume failure (pessimist!)
+        if ((o != null) && o instanceof LayoutConnectivity) {
+            LayoutConnectivity lc = (LayoutConnectivity) o;
+            result = toString().equals(lc.toString());
+        }
+        return result;
+    }
 
-}
+    @Override
+    public int hashCode() {
+        return toString().hashCode();
```

Comment:
It is better to add the hashCodes of the fields of this class to make the hashCode than to to use the hashCode of the results of toString().